cmd/jaeger/config-elasticsearch.yaml

```diff
@@ -32,6 +32,11 @@ extensions:
     backends:
       some_storage:
         elasticsearch:
+        server_urls: "http://your es ip:9200"
+          auth:
+            basic:
+              username: "your es username"
+              password: "your es password"
```

Comment:
this will break our CI, we validate this config file in our e2e tests. I am not opposed to this change (we did a similar change to enable password for Cassandra), but the auth needs to be enabled in ES server